jax/numpy/__init__.pyi

```diff
@@ -181,8 +184,15 @@ def ceil(x: ArrayLike, /) -> Array: ...
 character = _np.character
 def choose(a: ArrayLike, choices: Sequence[ArrayLike],
            out: None = ..., mode: str = ...) -> Array: ...
-def clip(a: ArrayLike, a_min: Optional[ArrayLike] = ...,
-         a_max: Optional[ArrayLike] = ..., out: None = ...) -> Array: ...
+def clip(
+    x: ArrayLike,
```

Comment:
I think `x` needs a default value, otherwise existing uses of `jnp.clip(a=array, a_min=0, a_max=1)` will error.

Reply:
Good catch -- added `None` as a default value for now, including an additional check to ensure that `jnp.clip(min=..., max=...)` throws an obvious error.